tests/install/features/installation.feature

```diff
@@ -0,0 +1,10 @@
+Feature: MetalK8s installation
```

Comment:
I guess we will have various scenarios for this feature (single-node, multi-node,...). How can we make sure all of them run in a separate environment/VM?

Reply:
Ignore. In fact parametrized over an inventory.